tools/setup_permissions.sh

```diff
@@ -46,6 +46,15 @@ function grant_task_execution() {
          -H "Authorization: token=${ACS_TOKEN}" \
          "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:master:task:user:${LINUX_USER}/users/${SERVICE_ACCOUNT_NAME}/create"
 
+    # XXX 1.10 curerrently requires this mesos:agent permission as well as
+    # mesos:task permission.  unclear if this will be ongoing requirement.
+    # See DCOS-15682
+    curl -k -L -X PUT \
+         -H "Authorization: token=${ACS_TOKEN}" \
+         "${DCOS_URL}/acs/api/v1/acls/dcos:mesos:agent:task:user:${LINUX_USER}" \
```

Comment:
I didn't mean to explicitly use ${framework_name}, the point I was making is that this script needs to also add the "similar curl" to grant the access control action to the service account (...-principal).